src/components/POIForm.component.js

```diff
@@ -187,4 +170,46 @@ class POIForm extends Component {
   }
 }
 
+function FieldGroup({
+  controlId, //indentifier
+  label,
+  inputType, //text vs. textarea vs. file
+  helperText,
+  thisValue,
+  thisChange
```

Comment:
Both of these can just be called `value` and `onChange`. `thisValue` and `thisChange` might get confusing because "this" is context specific